modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -47,6 +48,23 @@ kubernetes-nmstate-operator.v4.18.0   Kubernetes NMState Operator   4.18.0
 $ oc delete --namespace openshift-nmstate clusterserviceversion kubernetes-nmstate-operator.v4.18.0
 ----
 
+. Before you delete the `nmstate` CR, remove the `nmstate-console-plugin` plugin resource from the `nmstate` CR so that `nmstate-console-plugin` does not persist.
++
+.. Create an environment variable that uses the `jq` CLI to find the location of `nmstate-console-plugin` among the list of available plugins. The following example assigns a command to an environment variable named `INDEX`:
```

Comment:
The following example assigns a command to an environment variable named `INDEX` -->

The following **command** assigns the index of the `nmstate-console-plugin` entry in the `plugins` array to an environment variable named `INDEX`

Reply:
Very good :+1: